tools/uswds-component-library-figma.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+title: USWDS Component Library in Figma
+nav: Tools
+permalink: /tools/uswds-component-library-figma
+sidenav: tools-side-nav
+---
+
+# {{ page.title }}
+
+The [USWDS component library](https://www.figma.com/community/file/836611771720754351) is available and free to use through the Figma Community. Created by the [Truss.works team](https://truss.works/), it's a great place to start incorporating components into your designs.
+
+## Getting started
+
+### Duplicating the USWDS component library
+
+To view and duplicate files from the Figma Community, login or create a [Figma account](https://www.figma.com/).
+
+1. Navigate to the community tab in the left-hand side navigation
```

Comment:
I realize the style guide is only in my head right now, but let's add periods to these list items.